packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -1,10 +1,21 @@
 import PouchDB from "pouchdb";
 import PouchDBMemoryAdapter from "pouchdb-adapter-memory";
 import PouchDBFind from "pouchdb-find";
-
+import pouchdbDebug from "pouchdb-debug"
 import { soliditySha3 } from "web3-utils";
+import jsonStableStringify from 'json-stable-stringify';
+
+type PouchApi = {
+  bytecodes?: PouchDB.Database,
```

Comment:
I'm wondering why the `?` here -- I feel like each resource's db should be required, and they all get generated when we load up Pouch in this file.

Reply:
I agree with you here. Thanks for pushing for this. I modified the code to make PouchApi stricter so we can have type checks in the future; and explicitly casted the initialization `as` a PouchApi to declare the object is a PouchApi.